File: igor/Problema0369/problema0369.py

w, h = 5, 5
x = [[0 for i in range(w)] for j in range(h)] 

for i in range(0,w):
    for j in range(0,h):
        x[i][j] = int(input(": "))


m = 0

for i in range(0,w):
    for j in range(0,h):
        m = max(m, x[i][j])

f = [0]*(m+1)


for i in range(0,w):
    for j in range(0,h):
        f[x[i][j]] = f[x[i][j]]+1

for i in range(0,m+1):
    if f[i] > 1:
        print(i,": ",f[i])

File: igor/Problema0369/problema0369.java
import java.util.Scanner;



public class problema0369 {
    static Scanner input = new Scanner(System.in);
    public static void main(String[] args) {


        int x[][] = new int [5][5];

        for(int i=0;i<x.length;i++){
            for(int j=0;j<x.length;j++){
                System.out.println(i+" "+j+": ");
                int y = input.nextInt();
                x[i][j] = y;
            }
        }

        int m = 0;

        for(int i=0;i<x.length;i++){
            for(int j=0;j<x.length;j++){
                m = Math.max(m, x[i][j]);
            }
        }

        int[] f = new int[m + 1];

        
        for(int i=0;i<x.length;i++){
            for(int j=0;j<x.length;j++){
                f[x[i][j]]++;
                
            }
        }

        
        for (int i = 0; i < m + 1; i++){
            if (f[i] > 1) {
                System.out.println(i + ": "+f[i]);
            }
        }
    }
}